frontend/src/studio/components/sidebar/TrackControlsSidebar.tsx

```
@@ -240,6 +252,16 @@ const TrackControls: React.FC<TrackControlsProps> = ({
         );
         
       case 'audio':
+      case 'sampler':
+        // Explicitly cast track to the correct type to satisfy TypeScript
+        const audioTrack = track as AudioTrackState;
```

Biome (1.9.4)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Unsafe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)